prody/atomic/atomgroup.py

```diff
@@ -371,6 +394,11 @@ def __add__(self, other):
 
         return new
 
+    def extend(self, other):
```

Comment:
I don't think this should be necessary because people can just use add with newAG=False

Reply:
yes I agree